src/acvp_build_register.c

```diff
@@ -2003,40 +1972,40 @@ static ACVP_RESULT acvp_build_kas_ecc_register_cap (ACVP_CTX *ctx, JSON_Object *
                     curve = current_pset->curve;
                     switch (curve)
                     {
-                    case ACVP_ECDSA_CURVE_B233:
+                    case ACVP_EC_CURVE_B233:
```

Comment:
Could this switch be replaced with acvp_lookup_ec_curve_name() too?

Reply:
yes it sure can be! I'll make the change